iocore/eventsystem/I_SocketManager.h

```diff
@@ -102,7 +110,7 @@ struct SocketManager {
   int dup(int s);
 
   // result is the fd or -errno
-  int accept(int s, struct sockaddr *addr, socklen_t *addrlen);
+  int accept4(int s, struct sockaddr *addr, socklen_t *addrlen, int flags = 0);
```

Comment:
I was on the fence about this. I went with the new name because we have a lot of `accept()` functions floating around and I wanted the change to be clear. Also, since we are bringing in the new API by emulating `accept4()` on platforms where it didn't exist, I figured the name change made sense. Do any others have an opinion about this? @jpeach? @zwoop?